launch/dynamo-run/src/input/text.rs

```diff
@@ -122,7 +122,13 @@ async fn main_loop(
         };
 
         // Call the model
-        let mut stream = engine.generate(Context::new(req)).await?;
+        let mut stream = match engine.generate(Context::new(req)).await {
+            Ok(stream) => stream,
+            Err(err) => {
+                tracing::error!(%err, "Request failed.");
```

Comment:
I think its maybe difficult to know if this is a temporary failure or not .... I think this is a not a bad incremental step - but I think we do need to improve further to indicate permanent vs temporary failure - warnings .... 

backend not starting is kind of hidden at this point -